umh.docs.umh.app/content/en/docs/features/umh-lite.md

```diff
@@ -0,0 +1,91 @@
+---
+title: UMH Lite
+menuTitle: UMH Lite
+description: Understand the purpose and features of the UMH Lite, as well as the differences between UMH Lite and UMH Classic.
+weight: 2000
+edition: community
+aliases:
+
+---
+
+The UMH Lite is a stripped-down version of the classic United Manufacturing Hub.
+It allows you to extract data from your shop floor and send it to your own data
+infrastructure, as well as perform network monitoring.
+
+## When should I use it?
+
+If the UMH Classic seems a bit much and all you currently need is a way to get
+your data from the shop floor to your existing data infrastructure, you are in
+the right place. The UMH Lite allows you to gather data and send it to your own
+data infrastructure. It does not offer most of the features of the UMH Classic:
+you will not install Grafana or Kafka, and you will not have a Unified
+Namespace. However, you will be able to monitor the connection health of your
+network and spot errors with a single glance.
+
+## What can I do with it?
+
+### Differences between UMH Lite and Classic
+
+The UMH Lite does not offer all features of the UMH Classic. In the table below,
+you can find a comparison between both, and check if the UMH Lite fulfills your
+needs.
+
+| Feature                   | Classic | Lite |
+|---------------------------|:-------:|:----:|
+| Historian                 |    ✓    |      |
+| Analytics                 |    ✓    |      |
+| Data Visualization        |    ✓    |      |
+| UNS                       |    ✓    |      |
```

Comment:
UNS and MQTT+kafka are I think the same thing (or could be understand as the same thing)

Reply:
Yes, but I thought it would be beneficial to state it explicitly since users are often deterred by Kafka. I will change it to UNS (Kafka & MQTT).